### PR TITLE
Updates to CCH and GCMRC descriptions

### PR DESCRIPTION
**Pull request by mhines-usgs**

### CODE REVIEW

File: data/gaTable.yaml [hunk 1]
```diff
@@ -567,7 +567,7 @@
   analyticsContact: 'David Watkins'
   analyticsContactEmail: 'wwatkins@usgs.gov'
 
-- longName: Grand Canyon Monitoring Research Center Website
+- longName: Grand Canyon Monitoring and Research Center''s Discharge, Sediment, and Water Quality Monitoring App
```
Comment (mhines-usgs) [inline data/gaTable.yaml]: I wasn't sure if I had to escape the single apostrophes -- if not, nope! ;)

Comment (mhines-usgs) [inline data/gaTable.yaml]: I see another typo I made, if you can confirm I'll put another PR up without that. Thanks!

Comment (aappling-usgs) [inline data/gaTable.yaml]: i just tried out `y <- yaml::yaml.load_file('data/gaTable.yaml')`. with just one `'`, no escaping, it looks fine to me:
```

- longName: Grand Canyon Monitoring and Research Center's Discharge, Sediment, and Water Quality Monitoring App
  shortName: GCMRC
  accountName: CIDA GCMRC Analytics
  viewID: '80089136'
```

```
y <- yaml::yaml.load_file('data/gaTable.yaml')
y[[27]]$longName
## [1] "Grand Canyon Monitoring and Research Center's Discharge, Sediment, and Water Quality Monitoring App"
```

Comment (mhines-usgs) [inline data/gaTable.yaml]: Thanks @aappling-usgs !

**Pull request merged**